app/assets/javascripts/shared/inline_editing.js

```diff
@@ -63,21 +74,26 @@ function initInlineEditing(title) {
       return true;
     }
 
-    $editBlock.click(e => {
+    $editBlock.click((e) => {
+      if (e.target.tagName === 'A') return true;
       if (inputString.disabled) {
         saveAllEditFields();
         editBlock.dataset.editMode = 1;
         inputString.disabled = false;
+        $inputString.removeClass('hidden').prev().addClass('hidden');
```

Comment:
All this `prev` without selector is calling for some troubles down the road (not only here). Would it be possible to add selector inside so it's more readable for someone what this is doing, or maybe also add a comment

Reply:
Move it to another line with selector